downloads/swan-lake-release-notes/swan-lake-2201.7.0/RELEASE_NOTE.md

```diff
@@ -82,6 +100,34 @@ public function main() {
 
 ### Improvements
 
+#### Support for constant declarations without a type descriptor
+
+Constants can now be declared without specifying a type descriptor. Previously it was required to specify a type descriptor except with numeric and string literal expressions.
```

Comment:
```suggestion
- Constants can now be declared without specifying a type descriptor. Previously it was required to specify a type descriptor except with numeric and string literal expressions.
```

Reply:
These are descriptions under the topic. Don't have to be bullet points.

Reply:
The bullet points were added to preserve consistency with the other sections as per[1].

@anupama-pathirage - WDYT?

[1] https://ballerina.io/downloads/swan-lake-release-notes/swan-lake-2201.6.0#new-langintavg-function

Reply:
One reason to add a bullet even though we have a single point is that if not, the standard library section will show that inconsistency highlighted a lot with these one-liners below.

<img width="867" alt="Screenshot 2023-06-16 at 20 31 36" src="https://github.com/ballerina-platform/ballerina-dev-website/assets/11707273/ae4fe41e-7c06-48c0-96a1-4b0ad26f7bdd">

Reply:
I have updated the code as you suggested @praneesha. If we are reverting it raise it here

Reply:
I don't see a real value of adding a bullet points also. For standard library it makes sense as we always have very short sentences and not much code snippets or lengthy paragraphs. Here in compiler section it is different. If we think about consistency why not adding it to `Introduction of the group by and collect clauses` too? But to me adding a bullet there seems bit unnecessary.

Reply:
+1 let's remove the bullet points.